src/Book.js

```diff
@@ -0,0 +1,24 @@
+import {useState} from 'react';
+import './Book.css';
+import BookCheckBox from './BookCheckBox';
+
+const Book = props => {
```

Comment:
Suggest using Card entity. Better do not use sbj scope at the component code if possible. Then if re-use - the sense won't be lost.